packages/block-library/src/navigation/index.php

```diff
@@ -291,9 +296,8 @@ function register_block_core_navigation() {
 					'default' => false,
 				),
 			),
-
-			'render_callback' => 'render_block_navigation',
 		)
 	);
 }
 add_action( 'init', 'register_block_core_navigation' );
+add_filter( 'render_block', 'render_block_navigation', 10, 2 );
```

Comment:
I don't know if the convention is codified anywhere, but typically I would expect to see the `add_filter` to be placed immediately following the callback function (`render_block_navigation` in this case).